bees/jenkinsbee/jenkinsbee.go

```diff
@@ -28,8 +28,9 @@ import (
 	"io/ioutil"
 	"net/http"
 	"time"
-
+	//"github.com/bndr/gojenkins"
```

Comment:
If we don't use it yet, remove this line or add a comment like "worth investigating and noteworthy for future updates".